include/open62541/architecture_definitions.h

```diff
@@ -415,11 +415,17 @@ UA_STATIC_ASSERT(sizeof(bool) == 1, cannot_overlay_integers_with_large_bool);
     #define UA_atomic_sync()
 #endif
 
+#if UA_MULTITHREADING >= 100
+#ifdef _MSC_VER // Visual Studio
+    #include <winsock2.h>
+#endif
+#endif
```

Comment:
Your master branch contains the following code which also seems to work here (at least for a short test, I might want to check a bit more thoroughly):
```suggestion
#if UA_MULTITHREADING >= 100 && defined(_WIN32)
#ifndef _WINSOCKAPI_
#define _NO_WINSOCKAPI_
#endif
#define _WINSOCKAPI_
#include <windows.h>
#ifdef _NO_WINSOCKAPI_
#undef _WINSOCKAPI_
#endif
#endif
```

Reply:
Yes, by via is a typo.